students/web-recuperada.html

```diff
@@ -0,0 +1,213 @@
+        <td> 931 GB </td>
+            
+        </tr>
+
+        <tr>
+        
+        <td> 1500 GB </td>
+        <td> 1396 GB </td>
+            
+        </tr>
+
+        <tr>
+        
+        <td> 2000 GB </td>
+        <td> 1862 GB </td>
+            
+        </tr>
+
+        <tr>
+        
+        <td> 3000 GB </td>
+        <td> 2793 GB </td>
+            
+        </tr>
+
+    </table>
+
+
+
+    <div>
+        <label for="username">Nombre:</label>
+        <input type="text" id="username" name="username">
+    </div>
+    
+    <div>
+        <label for="pass">Contraseña:</label>
+        <input type="password" id="pass" name="password"
+               minlength="8" required>
+    </div>
+    
+    <div>
+    <input type="submit" value="Entrar">
+            
+    
+    <label for="start">Fecha de nacimiento:</label>
+
+    <input type="date" id="start" name="trip-start"
+           value="2018-07-22"
+           min="2018-01-01" max="2018-12-31">  
+    </div>        
+            
+<div>            
+<label for="email">Inserte su e-mail aquí:</label>
+
+           <input type="email" id="email"
+                  pattern=".+@globex\.com" size="30" required>
+</div>    
+              
+<div>                  
+<p>Seleccione su género aquí:</p>
+
+            <div>
+                <input type="checkbox" id="masculino" name="masculino"
+                           checked>
+                    <label for="masculino">Maculino</label>
+                  </div>
+                  
+                  <div>
+                    <input type="checkbox" id="femenino" name="femenino">
+                    <label for="femenino">Femenino</label>
+                  </div>                  
+
+                  
+                  <p>Seleccione su género aquí:</p>
+
+                  <div>
+                    <input type="radio" id="femenino" name="drone" value="femenino"
+                           checked>
+                    <label for="femenino">femenino</label>
+                  </div>
+                  
+                  <div>
+                    <input type="radio" id="masculino" name="drone" value="masculino">
+                    <label for="masculino">masculino</label>
+                  </div>
+                  
+                
+                  <p>Tu color favorito:</p>
+
+                  <div>
+                      <input type="color" id="color" name="color"
+                             value="#e66465">
+                      <label for="color">Color</label>
+                  </div>
+                </div>
+
+<div>                
+<label for="avatar">Subir un fichero:</label>
+
+    <input type="file"
+    id="avatar" name="avatar"
+    accept="image/png, image/jpeg">
+
+    <label for="país">Elige un país:</label>
+    <select  name="país" id="país">
+        <option value="país">país</option>
+        <option value="turquía">turquía</option>
+      </optgroup>
+        <option value="japón">japón</option>
```

Comment:
remove accent or ñ or any character different of English for value option. It must be something like `<option value="japon">japón</option>`